Calibration/HcalCalibAlgos/test/RecAnalyzerHF.cc

```diff
@@ -197,9 +197,10 @@ void RecAnalyzerHF::endJob() {
   if (fillTree_) {
     cells = 0;
     for (auto itr : myMap_) {
```

Comment:
I know, it has not been changed in this PR, but I suggest to modify this to
```
for (const auto& itr: myMap_) {
```
to avoid the copy of the `std::pair`.